### PR TITLE
Update steam

### PR DESCRIPTION
**Pull request by rootmelo92118**

### CODE REVIEW

File: data/steam [hunk 1]
```diff
@@ -3,11 +3,13 @@ s.team
 steam-chat.com
 steamcommunity.com
 steamcontent.com
+steamdeck.com
 steamgames.com
 steampowered.com
 steamstatic.com
 steamusercontent.com
 underlords.com
+valve.net
```
Comment (IceCodeNew) [inline data/steam:12]: can you provide some info about sites under the `valve.net` domain?

Comment (KevinZonda) [inline data/steam:12]: I double checked this domain, and only find one with ssl subdomain which will guide to a login form. The SSL certification shows it has some relationship with valve corp, however this is not kind of strong evidence I belive.

**Review by KevinZonda — CHANGES_REQUESTED** [review]

File: data/steam (hunk 1 above)
Comment (rootmelo92118) [reply data/steam:12]: Sorry, this domain was expired. I will remove that immediately.

**Comment by rootmelo92118** [comment]
Sorry, this domain only uses mail service. I will remove that immediately.

**Review by IceCodeNew — APPROVED** [review]

**Comment by IceCodeNew** [comment]
Thanks for your contribution!

**Pull request merged**